website/common/script/cron.js

```diff
@@ -110,11 +110,9 @@ export function shouldDo (day, dailyTask, options = {}) {
   }
 
   let daysOfTheWeek = [];
-
   if (dailyTask.repeat) {
     for (let [repeatDay, active] of Object.entries(dailyTask.repeat)) {
-      if (!isFinite(DAY_MAPPING_STRING_TO_NUMBER[repeatDay])) continue; // eslint-disable-line no-continue
-      if (active) daysOfTheWeek.push(parseInt(DAY_MAPPING_STRING_TO_NUMBER[repeatDay], 10));
+      if (active && !isNaN(DAY_MAPPING_STRING_TO_NUMBER[repeatDay])) daysOfTheWeek.push(parseInt(DAY_MAPPING_STRING_TO_NUMBER[repeatDay], 10));
```

Comment:
For some reason, the Object.entries() function was mapping additional functions/properties for dailyTask.repeat, which pushed unnecessary values to daysOfTheWeek. I was using the daysOfTheWeek.length property later on.

I don't have the output handy, but I can generate it again for you if you'd like in a few.